### DISCUSSION TITLE
Wrong released files?

### DISCUSSION THREAD

**Discussion by edwinyzh** [pawelsalawa/sqlitestudio#3629] [Q&A]
Hello, 

Thanks for your great job! I was exciting for the new releases posted in recent days, but it seems that the wrong **old files** were included in the released zip files?

For example, attached is a screenshot of the files included in the [3.2.1 Critical fixes for initial 3.2 release] (https://github.com/pawelsalawa/sqlitestudio/releases/download/3.2.1/SQLiteStudio-3.2.1.zip) posted on Dec 31, 2019, the modification date of sqlitestudio.exe is July 2019. 
Is anything wrong? thanks.

![2020-01-02_12h18_24](https://user-images.githubusercontent.com/541197/71651666-5aafd200-2d5a-11ea-9284-8ee72ced1c9e.png)

**Comment by pawelsalawa** [comment]
The 3.2.1 version is relatively old - as you noticed. Version 3.2.2 is currently being worked on.

The "new files" you noticed is just moving old files from previous file hosting to the GitHub releases page.

**Comment by edwinyzh** [comment]
I see now, I thought it's new release :D
